exercises/hello-world/description.md

```diff
@@ -1,46 +1,11 @@
 ["Hello, World!"](http://en.wikipedia.org/wiki/%22Hello,_world!%22_program) is
-the traditional first program for beginning programming in a new language.
+the traditional first program for beginning programming in a new language
+or environment.
 
-**Note:** You can skip this exercise by running:
+The objectives are simple:
 
-    exercism skip $TRACK_ID hello-world
+- Write a function that says "Hello, World!".
+- Run the tests and make sure that they succeed.
+- Submit your solution and check it at the website.
 
-## Specification
-
-Write a `Hello World!` function that can greet someone given their name.  The
-function should return the appropriate greeting.
-
-For an input of "Alice", the response should be "Hello, Alice!".
-
-If a name is not given, the response should be "Hello, World!"
-
-## Test-Driven Development
```

Comment:
I agree that the instrutions were misplaced. Languages can use the proposed hint for their next exercise, whatever it may be.